<launch>
    <node name="safety_node" pkg="seabot_safety" type="safety_node" output="screen" ns="safety" respawn="true" respawn_delay="5">
        <param name="frequency" value="1.0" />

        <param name="safety_battery" value="true"/>
        <param name="safety_pressure_limit" value="true"/>
        <param name="safety_depressure" value="true"/>

        <param name="max_depth_reset_zero" value="1.0" />
        <param name="max_speed_reset_zero" value="0.01" />

        <param name="pressure_limit" value="6.2" />
        <param name="time_before_pressure_emergency" value="3.0" />

        <param name="enable_flash" value="true"/>
        <param name="limit_depth_flash_enable" value="0.5"/>

        <param name="battery_limit" value="10.0" />

        <param name="time_delay_batteries_msg" value="300000000000000000.0" />
        <param name="time_delay_internal_sensor_msg" value="2.0" />
        <param name="time_delay_external_sensor_msg" value="2.0" />
        <param name="time_delay_depth_msg" value="2.0" />
        <param name="time_delay_piston_state_msg" value="2.0" />
        <param name="time_delay_euler_msg" value="100000000000000000000.0" />

        <param name="delta_volume_allowed" value="1.0" />
        <param name="delta_ref_allowed" value="10.0" />
        <param name="volume_ref" value="0.006" />
        <param name="transition_tick_law" value="650.0" />
        <param name="pressure_internal_max" value="8500.0" /> <!-- a changer -->

        <param name="humidity_limit" value="70.0" />

								<param name="time_before_seafloor_emergency" value="30.0" />

    </node>
</launch>

